### DISCUSSION TITLE
bug: WalletConnectConnector does not respect FallbackProvider from configureChains

### DISCUSSION THREAD

**Discussion by nhardy** [wagmi-dev/references#170] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I noticed today that when the Polygon Mumbai public/default RPC (https://matic-mumbai.chainstacklabs.com/) was down, despite configuring higher priority RPC nodes with [`configureChains`](https://wagmi.sh/react/providers/configuring-chains), the failing public RPC node was the only node that was contacted for estimating gas or sending transactions.

### Expected Behavior

The connector should either:
- respect the FallbackProvider created by [`configureChains`](https://wagmi.sh/react/providers/configuring-chains)
- provide a method for specifying fallback RPC nodes
- provide a method for specifying a single RPC node

**Comment by jxom** [comment]
This is a bit tricky as you can only pass one RPC URL per chain to WalletConnect, and I don't think WalletConnect's EIP-1193 Provider has fallback functionality. 

Let's move this to discussions and we can keep track of it there.
